geoh5py/objects/text.py

```diff
@@ -0,0 +1,225 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2025 Mira Geoscience Ltd.                                     '
+#                                                                              '
+#  This file is part of geoh5py.                                               '
+#                                                                              '
+#  geoh5py is free software: you can redistribute it and/or modify             '
+#  it under the terms of the GNU Lesser General Public License as published by '
+#  the Free Software Foundation, either version 3 of the License, or           '
+#  (at your option) any later version.                                         '
+#                                                                              '
+#  geoh5py is distributed in the hope that it will be useful,                  '
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of              '
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the               '
+#  GNU Lesser General Public License for more details.                         '
+#                                                                              '
+#  You should have received a copy of the GNU Lesser General Public License    '
+#  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.           '
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+
+from __future__ import annotations
+
+import uuid
+from json import loads
+from typing import TYPE_CHECKING
+
+import numpy as np
+from pydantic import AliasChoices, BaseModel, Field, field_validator
+
+from .points import Points
+
+
+if TYPE_CHECKING:
+    from numpy import ndarray
+
+
+class TextEntry(BaseModel):
+    """
+    Core parameters for text mesh data.
+    """
+
+    starting_point: str = Field(
```

Comment:
Pydantic has an alias generator to handle this kind of aliasing.  From our forms class:
![image](https://github.com/user-attachments/assets/d7f208bc-b3d4-4436-b848-446e8baaeb96)

Reply:
Yep, brilliant